### DISCUSSION TITLE
PWAs no longer have custom taskbar icon, Windows 11 23H2.

### DISCUSSION THREAD

**Discussion by pklapperich** [filips123/PWAsForFirefox#474] [Q&A] [answered]
A few weeks back I upgrade Windows 11 Pro to 23H2 (build 22631.3155). Sometime in there I probably updated Firefox (123.0.1) and PWAsForFirefox (2.10.1 extension, 2.10.1 native, 123.0.1 runtime) as well. I didn't pay attention to when exactly it stopped working, but the taskbar icon for all of my PWAs is now the standard Firefox icon and not the custom icon defined by the PWA.

Anyone else having this trouble? Did MS break something in Win 11 23H2 or do I need to do some troubleshooting and figure out why mine is broken?

**Comment by filips123** [comment]
Check if editing/updating the PWA from the extension fixes the issue. Also, please obtain the [runtime logs](https://pwasforfirefox.filips.si/help/troubleshooting/#obtaining-runtime-logs).

**Reply by pklapperich** [reply, marked as answer]
I figured it out. I guess I changed my most commonly used PWA to custom SVG icons and those ones weren't working. Switching them to transparent PNGs worked.

**Reply by filips123** [reply]
Well, SVG icons should also work... So, what exactly have you done that caused this issue?

**Reply by filips123** [reply]
It seems SVG icons actually aren't properly supported on Windows. I will try to fix this in the future.

**Reply by pklapperich** [reply]
If you need a test image, one of the icons I was using is the old icon for [Google Chat from Wikipedia](https://en.wikipedia.org/wiki/Google_Chat#/media/File:Google_Chat_icon_(2020).svg), which I uploaded [somewhere else](https://gist.githubusercontent.com/pklapperich/2fc5216870ec1b5f7ff5fe1a5322918f/raw/67b88a2743974d9c81e8f7c01c826b8550dc7ea3/chat.svg) so as not to hotlink.
